libs/providers/growthbook-client/src/lib/growthbook-client-provider.ts

```diff
@@ -0,0 +1,82 @@
+import { EvaluationContext, Provider, JsonValue, ResolutionDetails, GeneralError } from '@openfeature/web-sdk';
+
+import { FeatureResult, GrowthBook, InitOptions, Context } from '@growthbook/growthbook';
+import isEmpty from 'lodash.isempty';
+
+export default function translateResult<T>(result: FeatureResult, defaultValue: T): ResolutionDetails<T> {
+  const resolution: ResolutionDetails<T> = {
+    value: result.value === null ? defaultValue : result.value,
+    reason: result.source,
```

Comment:
It looks like `source` can represent various error conditions. For example, I'm assuming `unknownFeature` means the SDK couldn't find the request feature flag. Could you please try and map those conditions to approprate `ErrorCode`?